app/models/annotation/Annotation.scala

```diff
@@ -444,18 +450,23 @@ class AnnotationDAO @Inject()(sqlClient: SqlClient, annotationLayerDAO: Annotati
     val stateQuery = getStateQuery(isFinished)
     for {
       accessQuery <- baseListAccessQ
-      rows <- run(q"""select count(_id) from $existingCollectionName
-                   where typ = ${AnnotationType.Explorational} and ($stateQuery) and ($accessQuery)""".as[Long])
+      rows <- run(q"""SELECT COUNT(*)
+                      FROM $existingCollectionName
+                      WHERE typ = ${AnnotationType.Explorational}
+                      AND $stateQuery
+                      AND $accessQuery""".as[Long])
```

Comment:
Yes, string interpolation in scala only needs the brackets if the inserted expressions contain special characters like `.` – simple camelCase variable names can and should be used with a bare `$`.

Reply:
Oh, I thought that the brackets were supports to be within the SQL query and should be interpreted by Postgres so that the expressions `$stateQuery` and `$accessQuery` might not break operator precedence / the query syntax. :thinking: 

Scala needing the brackets around / before the `$` seems kinda counterintuitive and reads to me like the bracket would end up in the formatted string :man_shrugging: